package.json

```diff
@@ -50,18 +50,20 @@
     "event-stream": "3.3.2",
     "express": "4.13.4",
     "fs-extra": "0.30.0",
+    "he": "^1.1.0",
```

Comment:
It's an encoder, but it's not particularly trivial. Especially given that this is a protective measure I'm far more comfortable relying on `he` than attempting to do it myself.